content/docs/user-guide/interactivity/user-interface/canvases/canvas-properties.md

```diff
@@ -12,44 +12,43 @@ The canvas properties are displayed in the **UI Editor** **Properties** pane whe
-+ **Navigation threshold** - The analog input value, from a thumbstick, for example, that must be exceeded before a navigation command is processed. Valid ranges are decimal values between 0 and 1, and the default is 0.4. Adjust this value based on the input sensitivity needs of your UI.
-+ **Navigation repeat delay** - The delay, in milliseconds, before a held navigation command begins repeating. The default is 300 ms. Adjust this value based on the needs of your UI.
-+ **Navigation repeat period** - The delay, in milliseconds, after the initial repeat delay, before a held navigation command repeats again. The default is 150 ms. Adjust this value based on the needs of your UI.
-
-  For example, if you had a menu list where you hold a button to navigate to the next item in the list, the navigation property settings are used as follows:
 
-  1. Hold down the button past the *navigation threshold* to navigate to the next item.
-
-  1. Continue holding for an amount of time equal to the *navigation repeat delay* to navigate a second time.
-
-  1. Continue holding for an amount of time equal to the *navigation repeat period* to navigate a third time. Thereafter, as you continue holding the button, you will navigate again, every time an amount of time equal to the navigation period elapses.
-+ **First focus element** - Displayed when **Handle navigation** is selected. **First focus element** specifies which element gains focus when a canvas is first loaded and a mouse is not detected. For more information about element navigation, see [First Focus Element](/docs/user-guide/interactivity/user-interface/components/interactive/properties/navigation/components-firstfocus).
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Handle positional** | Causes an automatic response to positional input such as mouse movement, mouse button clicks, and touch screen input. Keyboard inputs also cause an automatic response when an interactive UI element is active (such as an elemental with a **Text Input** component on it). <br><br> Common reasons to de-select this property are if the canvas doesn't require input, or if you configure your game to handle all inputs and pass selected inputs to the UI system.| Boolean | `True` |
+| **Consume all input** | If enabled, this canvas consumes *all* input events, regardless of whether the canvas handles a specific input event. For example, if you have canvas A covering canvas B, you probably don't want canvas B handling any input while canvas A is obstructing it, so you would select this property on canvas A. Modal dialog boxes are another example of a canvas that should have this property selected. <br><br> Note that anytime a canvas is loaded, if it's set to consume all input, then it steals the inputs from any other loaded canvas. This includes canvases that are set to consume all inputs themselves. | Boolean | `False` |
```

Comment:
honestly unsure if "anytime" is correct here, but they say "any time" is always correct.

```suggestion
| **Consume all input** | If enabled, this canvas consumes *all* input events, regardless of whether the canvas handles a specific input event. For example, if you have canvas A covering canvas B, you probably don't want canvas B handling any input while canvas A is obstructing it, so you would select this property on canvas A. Modal dialog boxes are another example of a canvas that should have this property selected. <br><br> Note that any time a canvas is loaded, if it's set to consume all input, then it steals the inputs from any other loaded canvas. This includes canvases that are set to consume all inputs themselves. | Boolean | `False` |
```

Reply:
"while" would also be a good choice here.